guide/misc/release-notes.md

```diff
@@ -14,15 +14,121 @@ to Brooklyn's commercial users for funding much of this development.
+* `ProxyEffector` forwards effector invocations from one entity to another.
+* `ConditionalEntity` creates an entity based on the truth of a config key
+  (e.g. `$brooklyn:scopeRoot().config("enable.loadBalancer")`).
+* `OnPublicNetworkEnricher` grants finer control of mapping public
+  and private network addresses to sensors.
+* `PercentageEnricher` publishes the ratio of two sensors.
+
+New blueprint DSL features:
+
+* `$brooklyn:entity` supports nested function calls,
+* `$brookln:getImmediately` resolves values immediately and throws if it cannot,
+* `$brooklyn:object` supports parameterised constructors and static factory methods, hugely increasing the range of objects that can be injected into entities,
+* `$brooklyn:entityId` returns the target entity's ID, and
+* `$brooklyn:self` accesses the resolving task's context entity.
+
+
+#### Bundled entities
+
+There have been many updates to the entities bundled with Brooklyn. In particular:
+
+* `DynamicCluster` allows configuration of member removal strategies.
+* The `Transformer` enricher can be triggered by events on multiple sensors.
+* `ZooKeeperNode` publishes main.uri and `ZooKeeperEnsemble` aggregates
+  hosts and ports into comma-separated list under `zookeeper.endpoints`.
+* `BindDnsServer` listens on all interfaces and supports Centos and RHEL.
+* `NginxController` supports proxying with TLS client certificates.
+* `RiakNode` optionally configures security groups for intra-node communication
+  when supported by the target cloud.
+* `PostgreSqlNode` can be configured to grant [user roles](https://www.postgresql.org/docs/9.3/static/role-membership.html).
+* `SharedLocationSecurityGroupCustomizer` enables security group 
+  configuration in yaml blueprints.
+* Feeds no longer poll if their entity is unmanaged.
+
+Several new test-case entities are introduced:
+
+* `TestHttpCall` [supports](https://github.com/apache/brooklyn-server/pull/414) a greatly expanded range of options.
+* `TestEndpointReachable` tests that a TCP endpoint can be reached.
+* `TestWinrmCommand` tests the success of commands in Windows instances.
+* `RelativeEntityTestCase` lets you write test cases that resolve their target from another entity.
+
+Additionally, location customisers can now be attached to entities as
+initialisers, rather than having to dig into their `templateOptions`.
+
+
+#### Brooklyn server
+
+* The REST API requires CSRF headers for non-GET and HEAD requests when
+  sessions are in effect.
+  See [CsrfTokenFilter](https://brooklyn.apache.org/v/0.10.0/misc/javadoc/org/apache/brooklyn/rest/filter/CsrfTokenFilter.html)
+  for details.
+
+* The catalogue scanner can be configured with whitelists and blacklists
+  of bundle IDs to control which bundles may present applications in the catalogue.
+
+* The Karaf distribution includes equivalents of the
+  [cloud explorer commands](https://brooklyn.apache.org/v/0.10.0/ops/server-cli-reference.html#cloud-explorer).
```

Comment:
Agree. The example I was referring to is:
```
{{ site.path.v }}/0.9.0/misc/release-notes.html
```
This makes it possible to have the docs live under a different root, possible a local one for development, testing. Or a staging area for the docs.